modules/phase_field/src/actions/NonconservedAction.C

```diff
@@ -97,7 +101,6 @@ NonconservedAction::act()
     InputParameters params1 = _factory.getValidParams(kernel_type);
     params1.set<NonlinearVariableName>("variable") = _var_name;
     params1.applyParameters(parameters());
-
```

Comment:
I'd prefer it if unnecessary white space changes we reduced to a minimum

Reply:
removed them from the first commit